Backend/app/rider-platform/rider-app/Main/spec/Storage/MultiModal.yaml

```
@@ -183,3 +200,92 @@
+    fromStopCode:  (fromStopDetails >>= (.stopCode))|E
+    fromStopName:  (fromStopDetails >>= (.name))|E
+    fromStopGtfsId: (fromStopDetails >>= (.gtfsId))|E
+    toStopCode:  (toStopDetails >>= (.stopCode))|E
+    toStopName:  (toStopDetails >>= (.name))|E
+    toStopGtfsId: (toStopDetails >>= (.gtfsId))|E
+    routeGtfsId: (routeDetails >>= (.gtfsId))|E
+    routeLongName: (routeDetails >>= (.longName))|E
+    routeShortName: (routeDetails >>= (.shortName))|E
+    routeColorName: (routeDetails >>= (.routeColorName))|E
+    routeColorCode: (routeDetails >>= (.routeColorCode))|E
+    frequency : (routeColor >>= (.frequency))|E
```

yamllint (1.35.1)

[warning] too many spaces before colon

(colons)